### PR TITLE
Maybe for the style guide again - Docs qtdragon2

### PR DESCRIPTION
**Pull request by smoe**
These "+"s do not show up in po4a-generated files, so admittedly I am in part working around po4a-associated difficulties. But here, I just think they are too much.

### CODE REVIEW

**Comment by smoe** [comment]
Darn, these pyvcp examples should have been in another PR. Anyway...

File: docs/src/gui/pyvcp-examples.adoc [hunk 1]
```diff
@@ -47,9 +47,9 @@ The -Wn makes HAL 'Wait for name' to be loaded before proceeding.
 
 The pyvcp -c makes PyVCP name the panel.
 
-The HAL pins from panel1.xml will be named btnpanel.<pin name>
+The HAL pins from panel1.xml will be named btnpanel.<_pin name_>.
```
Comment (hansu) [inline docs/src/gui/pyvcp-examples.adoc:50]: This doesn't work. Use `btnpanel._<pin name>_` instead.

**Comment by silopolis** [comment]
Le ven. 2 sept. 2022 à 12:34, Steffen Möller ***@***.***> a
écrit :

> These "+"s do not show up in po4a-generated files, so admittedly I am in
> part working around po4a-associated difficulties. But here, I just think
> they are too much.
>

Wasn't the forced line return issue fixed in po4a 0.67?

**Comment by silopolis** [comment]
Le ven. 2 sept. 2022 à 17:35, Steffen Möller ***@***.***> a
écrit :

> Not sure. We now have linebreaks in weblates that we did not have before.
> But we lost the '+'s.
>

As I hardly see how they could be reintroduced afterward, this means the
translated adoc files must be broke and that the issue is omehow still
there :/

Maybe this time it's a Weblate problem? 🤔

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> Not sure. We now have linebreaks in weblates that we did not have before.
> But we lost the '+'s.

[Jérémie Tarot]
> As I hardly see how they could be reintroduced afterward, this means the
> translated adoc files must be broke and that the issue is omehow still
> there :/
>
> Maybe this time it's a Weblate problem? 🤔

Nah, it is simply an issue of expectations.  Let me try again to explain
how this (correctly) work:

If the adoc file have "first line +\nsecond line", the POT file get
"first line\nsecond line", the PO file traslate to
"første linje\nandre linje", and the generated translated adoc file get
"første linje +\nandre linje".  Both adoc files are correct, and the
translators do not have to worry about placing SPACE PLUS NEWLINE in the
correct sequence, and only have to place the NEWLINE in the correct spot.

See
<URL: https://github.com/mquinson/po4a/commit/afe0544cfaad8d5d4ac3e6caf0529b69300d4aef >
and previos related patch alongside the automated test to validate the
behaviour.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
As a translator I prefer not to do anything but rely on the automated translation (that button click already annoys me) and do a quick confirmation that everything is translated as it should be ("ah, not that port -> harbor miss-translation again"). RSI and time are prohibiting everything else - no manual typing if at all possible. 
Now, when there are "verbatim" blocks, then
```
this
that
```
gets translated to just that. But the same appearing in another context, i.e. a regular paragraph or a note, means something different, I would have to manually enter the ' +' to transform
```
this
that
```
to
```
this +
that
```
since weblate does not see the +.

For most of our texts this is not problematic since there are no explicit linebreaks and for some texts there are far too many, so I am not too unhappy to see them negatively selected or you get an indication from reading the text where the + should go. But then again - argh!

I am pretty confident that the elimination of the "+" is a bug. Petter thinks differently. And I can live with what we have for now. There is another bug of po4a that removes the last character of table fields, as in a confusion of chomp with chop, but it is not perfectly reproducivle as it seems and I have not had the motivation to chase that up either, which should have a higher priority.

**Comment by smoe** [comment]
I just read @petterreinholdtsen's comment again. I see a loss of semantics if the + goes away. And I do not want to manually do anything. The '+' I found to be treated well by the automated translators, so I wish it would remain in. The newline to consider special white space (if I interpret this right) sounds inconvenient to me. So, for now, I think I will just continue avoiding the '+'s.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> I just read @petterreinholdtsen's comment again. I see a loss of
> semantics if the + goes away. And I do not want to manually do
> anything. The '+' I found to be treated well by the automated
> translators, so I wish it would remain in. The newline to consider
> special white space (if I interpret this right) sounds inconvenient to
> me. So, for now, I think I will just continue avoiding the '+'s.

I suspect either I misunderstand you, or you misunderstand me, as what
you write do not make sense to me.  The newline in adoc strings and a
lot of other formats in the PO is meaningful.  The plus sign (alone) is
not.

The current German translation of the documenation is not buildable (see
#1983), because of incorrect handling of newline.  Weblate have
automatic checks for this, and the German translation currently have 160
strings reported with "Failing check: Mismatching line breaks" in <URL:
https://hosted.weblate.org/projects/linuxcnc/linuxcnc-docs/de/ >,
alongside alot of other failing checks.  There are lots of other failing
checks too.  I strongly suggest to fix every one of them, either by
rewriting the string or by flagging them as ignored, to avoid new
problems sneaking into the translation.

-- 
Happy hacking
Petter Reinholdtsen

File: docs/src/gui/pyvcp-examples.adoc (hunk 1 above)
Comment (hansu) [reply docs/src/gui/pyvcp-examples.adoc:50]: \*dont forget me\*

**Comment by silopolis** [comment]
#1530 Style guide

File: docs/src/gui/pyvcp-examples.adoc (hunk 1 above)
Comment (hansu) [reply docs/src/gui/pyvcp-examples.adoc:50]: Ok then not italic...
